### PR TITLE
Add treasury-funded wallets

### PR DESCRIPTION
**Pull request by laboon**
DO NOT MERGE YET - NEEDS TO BE REVIEWED

Adds four Treasury-funded wallets/signers

### CODE REVIEW

File: docs/build/build-wallets.md [hunk 1]
```diff
@@ -39,3 +39,16 @@ the wallet software is using the latest version of the [Polkadot API](https://po
 > means that you are at risk if connected to a compromised version of Polkadot-JS. A fix is being
 > worked on; please see [this GitHub issue](https://github.com/paritytech/parity-signer/issues/724)
 > for more details.
+
+### Polkadot or Kusama Treasury Funded Wallets
```
Comment (salmad3) [inline docs/build/build-wallets.md]:
```suggestion
### Treasury Funded Wallets
```

File: docs/build/build-wallets.md [hunk 2]
```diff
@@ -39,3 +39,16 @@ the wallet software is using the latest version of the [Polkadot API](https://po
...
+| --------------------------------------------------------- | ----------------- | -------------- | ------------ | ------------- | -------- |
+| [Fearless](https://kusama.polkassembly.io/treasury/102)   | Live              | Soramitsu      | iOS, Android | Non-custodial | Staking  |
+| [Klever](https://kusama.polkassembly.io/treasury/91)      | Live              |                | iOS, Android | Non-custodial | Staking  |
+| [Polkawallet](https://kusama.polkassembly.io/treasury/41) | Live              |                | iOS, Android | Non-custodial | Staking  |
+| [Stylo](https://polkadot.polkassembly.io/treasury/39)     | Live              | Thibaut Sardan | Android      | Non-custodial | Staking  |
+
+> Note that these third-party wallets have been funded by the community through either the Polkadot or Kusama [Treasury](learn-treasury). Web3 Foundation does not endorse these wallets, and you should use your own due diligence in researching them. Polkadot Support cannot provide support for issues with these wallets or other non-Parity developed wallets.
```
Comment (salmad3) [inline docs/build/build-wallets.md]:
```suggestion
> NOTE: these third-party wallets have been funded by the community either through the Polkadot or Kusama [Treasury](learn-treasury). **Web3 Foundation does not endorse these wallets, and you should use your own due diligence in researching them.** The official Polkadot Support cannot provide support for issues with these wallets or other non-Parity developed wallets.
```

**Review by salmad3 — CHANGES_REQUESTED** [review]
If approved, here are some suggestions.
- With this update we can remove the first "Note"
```
> Note that only Parity-developed wallets are listed on this page. Other third-party wallets are
> available, but are not listed here.
```
-  Moving the following section under the "Supported Wallets" heading
```
[Polkadot support](https://support.polkadot.network/) can provide assistance with issues related to
Polkadot-JS, the Polkadot{.js} extension, or Parity Signer. For other wallet software, you should
contact the developers of that wallet.
```

- Suggesting to update headers:
 ```
 # Polkadot Wallets
 ## Supported Wallets
 ## Treasury Funded Wallets
 ## Encryption Enhancement 
 ```

**Comment by laboon** [comment]
Thanks Danny, I have accepted all of your suggestions (although since I force-pushed a new commit, I couldn't do it through the normal suggestions process)

**Comment by DrW3RK** [comment]
Looks good. Merging with master

**Pull request merged**